spring-cloud/src/main/java/com/solace/samples/cloudfoundry/springcloud/controller/SolaceController.java

```diff
@@ -117,12 +118,25 @@ public void init() {
 		
 		logger.info("Solace client initializing and using SolaceMessagingInfo: " + solaceMessagingServiceInfo);
 
+		String host = solaceMessagingServiceInfo.getSmfHost();
+		
```

Comment:
That is only relevant in the pojo sample. In Spring cloud you don't have to search for a ","